pygmt/src/pygmtlogo.py

```diff
@@ -0,0 +1,295 @@
+"""
+pygmtlogo - Create and plot the PyGMT logo.
+The design of the logo is kindly provided by `@sfrooti <https://github.com/sfrooti>`_.
+"""
+
+from pathlib import Path
+
+import pygmt
+
+
+def create_logo(  # noqa: PLR0915
+    blackwhite=False,
+    darkmode=False,
+    hexshape=False,
+    wordmark=True,
+    orientation="horizontal",
+):
```

Comment:
For the parameters, what about:

- `color`: `True` or color or `False` for black/white
- `theme`: `"light"` or "dark"
- `shape`: `"circle"` or `"hex"`
- `wordmark`: `True`/`False`/`"horizontal"`/`"vertical"` [True means "horizontal"].

Reply:
Including the `orientation` parameter into the `wordmark` parameter is a good idea!
Please see commit https://github.com/GenericMappingTools/pygmt/pull/3849/commits/b07fe8c9cdb6d679dc7fd668c0149eaff7b51c5e.

Reply:
`color` is shorter than `blackwhite`. Do people think there can be a confusion with the parameter `color` renamed to `fill` from the user side?

I used `bg_concept` (definitely not the best variable name) and `shape` at the beginning and I changed them to `darkmode` and `hexshape` (as hex is a built-in Python function) based on the comment by Michael (https://github.com/GenericMappingTools/pygmt/pull/3849#issuecomment-2721140604):

> The individual formats could be selected by different input parameters like hex = True or darkmode = True.

At the moment I am fine with both ways, using either `True` / `False` or strings.